src/dnf-plugins/subscription-manager.py

```diff
@@ -178,7 +178,7 @@ def _warn_or_give_usage_message():
             # Don't warn people to register if we see entitlements, but no identity:
             if not identity.is_valid() and len(ent_dir.list_valid()) == 0:
                 msg = not_registered_warning
-            elif len(ent_dir.list_valid()) == 0:
+            elif len(ent_dir.list_valid()) == 0 and not is_simple_content_access():
```

Comment:
The `identity` exists, Why not to use it as keyword argument in function `is_simple_content_access(identity=identity)`? We could potentially save some function calls.